### PR TITLE
1.0/284 create a pwa manifest

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name" : "ARC Voucher Recorder",
```
Comment (katjam) [inline src/manifest.json]: Do we need to [xXx] any of this copy?

**Review by katjam — APPROVED** [review]
Looks great unless we need to run the copy past anyone.

:orange_book: :panda_face:

File: src/manifest.json (hunk 1 above)
Comment (gingerbreadpup) [reply src/manifest.json]: Rose Voucher Recorder? ARC won't mean much to the traders.

**Review by gingerbreadpup — APPROVED** [review]
splendid! :shipit: :beach_umbrella:

**Pull request merged**